blog.md

```diff
@@ -0,0 +1,94 @@
+Here's a basic example:
+```html
+<html>
+  <head>
+    <title>Title of the Page</title>
+  </head>
+  <body>
+    <div id="base">
+      <a href="***">Link</a>
+      <p>Paragraph</p>
+    </div>
+    <table class="table">
+      <tr>
+        <th>Col 1</th>
+        <th>Col 2</th>
+      </tr>
+      <tr>
+        <td class="data">Row 1 Data 1</td>
+        <td class="data">Row 1 Data 2</td>
+      </tr>
+    </table>
+  </body>
+</html>
+```
+
+# Analyzing Our Website
+
+![Alt Text](https://d2p3bygnnzw9w3.cloudfront.net/req/202005291/logos/bbr-logo.svg)
+
+We will scrape statistics from a [set of](https://www.basketball-reference.com/play-index/psl_finder.cgi?request=1&match=single&type=per_game&per_minute_base=36&per_poss_base=100&lg_id=NBA&is_playoffs=N&year_min=1985&year_max=2020&franch_id=&season_start=1&season_end=-1&age_min=0&age_max=99&shoot_hand=&height_min=0&height_max=99&birth_country_is=Y&birth_country=&birth_state=&college_id=&draft_year=&is_active=&debut_yr_nba_start=&debut_yr_nba_end=&is_hof=&is_as=&as_comp=gt&as_val=0&award=&pos_is_g=Y&pos_is_gf=Y&pos_is_f=Y&pos_is_fg=Y&pos_is_fc=Y&pos_is_c=Y&pos_is_cf=Y&qual=&c1stat=pts_per_g&c1comp=gt&c1val=20&c2stat=&c2comp=&c2val=&c3stat=&c3comp=&c3val=&c4stat=&c4comp=&c4val=&c5stat=&c5comp=&c6mult=&c6stat=&order_by=pts_per_g&order_by_asc=&offset=0) basketball statistics to find players who scored 20 points per game or more since 1984. www.basketball-reference.com (a fantastic website) has a lot of query tools built into their website, but there filtering is limited, so we need to scrape the data to get what we want. To capture the data table, we first will right-click and select "Inspect Element" on Michael Jordan's name in the first row. As you hover over various parts of the code that pops up, you will see that there is a link (https://www.basketball-reference.com/players/j/jordami01.html) wrapped by a *td* set, and that there are pairs of *td* and *data-stat=* for each stat down the line like season, age, team_id, etc. These are wrapped by a *tr* pair for *data-row-=0*, which is preceded by a *tbody* pair. These are the relevant tags we will need to reference when scraping.
+
+![Alt Text](https://i.imgur.com/g8mi9vx.png)
+
+
+# Scraping the Data
+
+Understanding where our data lies is crucial for implementing the scraper. BeautifulSoup uses an html parser to locate the data we want, but we have to give it some baseline information to do so. 
+
+Before we start, we want to ensure we have the following libraries installed using pip install *package_name*: pandas, numpy, requests and bs4. The first thing we want to do for our scraper is determine how many pages of information there are. For this particular dataset, it is laid out across 9 different URLs. We have to examine what changes occur between two pages. It is a really long link, but we can see at the end there is a part that says offset=0 for the first page, but is equal to 100 for the second page, and increments by 100 each time. 
```

Comment:
consider an "installing dependencies" piece, either here or in the code with an inline comment about using pip3 (or another way?)